### PR TITLE
The issue-team.json was created

### LINKED ISSUE #17: Crear asset - team.json
### Descripción

Crear en la carpeta `assets/data` un archivo llamado `team.json` el cual contenga un array con la información de los organizadores de Codex en formato json.

### Criterios de aceptación

- La información de cada organizador debe tener la siguiente estructura:
```
{
  name: 'nombre y apellido',
  bio: 'texto',
  picture: 'enlace a foto de perfil de github',
  github: 'enlace al gitHub',
  twitter: 'enlace al twitter'
}
```

### Recursos:

- La información de los organizadores la puede encontrar en [este enlace](https://docs.google.com/spreadsheets/d/1ypNsnK5yoXDR_4suB4N8vouM9a9sKWGvOiaSN9XUyEc/edit?usp=sharing)

### Notas

- Puede dejar en blanco aquella información con la que no se cuente

### PR DESCRIPTION
**Pull request by jdiegoisaza**
## Description :black_nib:

the file team.json was created in the directory assets/data.

Fixes #17 

### Type of change :bomb:

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested? :page_with_curl:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### Checklist :white_check_mark:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### Impacted Areas in Application :triangular_flag_on_post:
List general components of the application that this PR will affect:

-

### CODE REVIEW

File: src/assets/data/team.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "team":[{
+      "name": "Martin Quintero",
+      "bio": "",
+      "picture": "",
```
Comment (ManuCastrillonM) [inline src/assets/data/team.json]: Recuerda agregar las fotos!
Por favor colocar los enlaces a las fotos de perfil de GitHub

Comment (ManuCastrillonM) [inline src/assets/data/team.json]: Recuerda agregar las fotos!
Por favor colocar los enlaces a las fotos de perfil de GitHub

File: src/assets/data/team.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "team":[{
```
Comment (MartinEliasQ) [inline src/assets/data/team.json]: Esta linea no es necesaria, la idea, es que el cuerpo del mensaje sea:
[{element1},{element2},{element3}]

Comment (MartinEliasQ) [inline src/assets/data/team.json]: Esta linea no es necesaria, la idea, es que el cuerpo del mensaje sea:
[{element1},{element2},{element3}]

File: src/assets/data/team.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      "name": "Manu Castrillón",
+      "bio": "Web Developer at Huge - Co-organizer of @pionerasdev",
+      "picture": "https://avatars3.githubusercontent.com/u/10585946?s=400&v=4",
+      "github": "https://www.github.com/manucastrillonm",
+      "twitter": "https://www.twitter.com/manucastrillonm"
+    }]
+    
+  }
```
Comment (MartinEliasQ) [inline src/assets/data/team.json]: Este signo de llave y el de la linea 1 no son necesarios.

**Comment by ManuCastrillonM** [comment]
Recuerda que el PR se debe hacer a la rama `develop`, porfa actualizalo

**Comment by ManuCastrillonM** [comment]
Recuerda que los mensajes de commit deben describir los cambios que hiciste, por favor modifica el mensaje de commit y resuelve los conflictos para poder hacer merge

**Pull request closed without merging**